docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+Continuous Patching Workflow in Azure Container Registry
+========================================================
+
+## Introduction
+
+Continuous Patching is a feature in Azure Container Registry that allows you to recurringly scan and patch specified artifacts for only OS-level  vulnerabilities within the registry. The workflow enables to set a cadence for run and create a recurring ACR task that scans your configured list of images for vulnerabilities (CVEs) using [Trivy](https://trivy.dev/) and patch them using [Copa](https://project-copacetic.github.io/copacetic/website/).
+ 
+
+> **NOTE:**
+> Continuous Patching is a limited pilot program as of October 2024.
+
+
+## Use Cases
+
+Here are a few scenarios to use Continuous Patching:
+
+- **Enforcing container security and hygiene:** Continuous Patching enables users to quickly fix OS container CVEs without the need to fully rebuild from upstream.
+- **Speed of Use:** Continuous Patching removes the dependency on upstream updates for specific images by updating packages automatically. Vulnerabilities can appear every day, while popular image publishers like Ubuntu only offer a new release once a month. With Continuous Patching, one can ensure that their image is OS vulnerability-free more frequently.
+
+## Preview Limitations
+
+Continuous Patching is currently in preview. The following limitations apply:
+- Windows-based container images aren’t supported
+- Only "OS-level" vulnerabilities will be patched. This includes packages in the image managed by a package manager such as “apt” and “yum”. Vulnerabilities at the “application level” are unable to be patched, such as compiled languages like Go, Python, NodeJS.    
+
+## Prerequisites        
+
+- You can use the Azure Cloud Shell or a local installation of the Azure CLI with a minimum version of 2.15.0 or later. 
+- You have an existing Resource Group with an Azure Container Registry.
+
+## Installing the Continuous Patching Workflow
+
+Download the [wheel file](https://acrcssc.z5.web.core.windows.net/acrcssc-1.1.1rc5-py3-none-any.whl) for Continuous patching 
```

Comment:
Customers who are not Python developers do not know of Python wheels. Instead, rephrase this to mention something like "private CLI extension"